projects/skia/skia.diff

```diff
@@ -0,0 +1,293 @@
+diff --git a/include/private/SkMalloc.h b/include/private/SkMalloc.h
```

Comment:
any reason why this isn't merged into skia? keeping diffs here might be a bit painful maintenance wise.

Reply:
I would like it in Skia as well, but I don't have buyoff to land it there yet.  I'm hoping I can prove value and let it live there permanently, but for now I'll pay the maintenance costs associated with it.